Dockerfile

```diff
@@ -18,7 +18,11 @@ COPY . /usr/local/src/cruton
 
 RUN apk -U add git && \
     rm /var/cache/apk/* && \
-    pip3 install --no-cache-dir --compile file:///usr/local/src/cruton#egg=cruton
+    pip install git+https://github.com/cloudnull/cruton
+#    pip install --compile file:///usr/local/src/cruton#egg=cruton
+
+
+ADD ./etc/cruton/cruton.ini /etc/cruton/cruton.ini
```

Comment:
I see this line which I was hoping would fix issue #2 but I pulled this p/r and still have the same issue:

````Traceback (most recent call last):
  File "/usr/local/bin/cruton-manage", line 10, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/site-packages/cruton/data_store/main.py", line 60, in main
    CONF(project='cruton')
  File "/usr/local/lib/python2.7/site-packages/oslo_config/cfg.py", line 2352, in __call__
    raise ConfigFilesNotFoundError(self._namespace._files_not_found)
oslo_config.cfg.ConfigFilesNotFoundError: Failed to find some config files: /etc/cruton/cruton.ini
```